features/unattached_commands.feature

```diff
@@ -313,3 +313,39 @@ Feature: Command behaviour when unattached
           | jammy   |
           # | kinetic | There is a very weird error on Kinetic, irrelevant to this test
           | lunar   |
+
+    @series.lts
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Verify autocomplete options
```

Comment:
Copy and paste fail? I assume this can be fixed later and doesn't need to block the SRU.

Reply:
Yep almost certainly a copy/paste error.
And yes, this doesn't affect the substance of the test (which is why no-one noticed until now :upside_down_face:)

Reply:
This was already fixed in a subsequent PR